rfcs/029-resource-notices.md

```diff
@@ -0,0 +1,101 @@
+- Feature Name: Resource Notices
+- Start Date: 2021-03-22
+- RFC PR: [amundsen-io/rfcs#29](https://github.com/amundsen-io/rfcs/pull/29) (after opening the RFC PR, update this with a link to it and update the file name)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000) (leave this empty for now)
+
+# Resource Notices
+
+## Summary
+
+Resource Notices allows Amundsen administrators to show informational, warning, and alert messages related to the different resources (tables, dashboards, people) we expose in Amundsen.
+
+## Motivation
+
+This feature will help administrators show messages related to deprecation, updates (or lack of), and informational messages related to specific resources. In Lyft's case, we use them to point final users to improved versions of current surfaced tables, so final users employ more trusted data.
+
+## Guide-level Explanation (aka Product Details)
+
+An Amundsen final user can see a notice in the table/dashboard/user detail page whenever their system admin adds one related to it.
+
+A notice is a small box with an icon and a message containing HTML markup (like links and bolded text).
+
+These will come in three flavors:
+
+- Informational: Marked with a blue "i" icon on the right side
+- Warning: Marked with an orange exclamation mark icon on the right side
+- Alert: Marked with a red exclamation mark icon on the right side
```

Comment:
Can you give an example or some semantics around each notice level?

Reply:
I think that will be up to the Amundsen admins to decide, we ourselves were doubting what to use for the ones we need to place right now.

Reply:
Let me ask this differently, how did we decide on three categories? Do we expect this list to change over time?

Reply:
We got feedback that having more types than the initial one (alert) would be useful. We agreed and introduced the basic severity pack (3 severities).

It could grow larger, maybe to 5 or 7, and it would be easy and backwards compatible if we decide to do so in the future.